Dockerfile

```diff
@@ -17,10 +17,10 @@ ARG DBNAME="data"
 ARG PORT=3000
 ARG CLUSTER_PORT=4000
 ARG HOST="0.0.0.0"
-ARG PATH="."
+ARG DBPATH="."
 
 # Set defaults for runtime environment variables, can be overriden by: docker run --env DBNAME=mydb ...
-ENV DBNAME=${DBNAME} HOST=${HOST} PORT=${PORT} CLUSTER_PORT=${CLUSTER_PORT} PATH=${PATH}
+ENV DBNAME=${DBNAME} HOST=${HOST} PORT=${PORT} CLUSTER_PORT=${CLUSTER_PORT} DBPATH=${DBPATH}
 
 # Start server without arguments, environment variables will be used
-CMD ["node", "start.js"]
\ No newline at end of file
+CMD ["node", "src/start.js"]
```

Comment:
`start.js` is within the `src` directory, which gets plopped into `/app/src/start.js` via line 13 (`COPY src ./src`), and the working directory (via line 10) is `/app`, so informing node of either `/app/src/start.js` or `src/start.js` makes sense to me :)

Reply:
Finer detail: `/app/start.js` is where the original line 26 says to look, but `start.js` doesn't exist there.